Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/) for Kodi add-ons: PVR clients/pvr.hts.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-pvr-clients/-/pvr-hts/horizontal-auto.svg)

ksooo commented on pvr.hts/resources/language/resource.language.de_de/strings.po:
```diff
@@ -60,6 +60,14 @@ msgctxt "#30010"
 
+msgctxt "#30011"
+msgid "Use HTTPS"
+msgstr "Use HTTPS"
```
This is definitely no German translation. Where does this string come from?

ksooo replied:
Reason is that the en_GB strings.po is not correct. We need to fix that.

gade01 replied:
Yeah, it needs to be blank here: https://github.com/kodi-pvr/pvr.hts/blob/Matrix/pvr.hts/resources/language/resource.language.en_gb/strings.po#L67

ksooo commented:
do not merge.

Jalle19 commented:
@ksooo should this still not be merged?

gade01 reviewed (approved):
Everything looks very good to me.

- Some new translations (no replacements)
- Some translation fixes done by me
- New strings

weblate committed:
Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (90 of 90 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (90 of 90 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 94.4% (85 of 90 strings)

Translated using Weblate (Sinhala (Sri Lanka) (si_lk))

Currently translated at 11.1% (10 of 90 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 95.5% (86 of 90 strings)

Translated using Weblate (Icelandic (is_is))

Currently translated at 94.4% (85 of 90 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 94.4% (85 of 90 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 94.4% (85 of 90 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 94.4% (85 of 90 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 37.7% (34 of 90 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (90 of 90 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 13.3% (12 of 90 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 37.7% (34 of 90 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Added translation using Weblate (Malayalam (India) (ml_in))

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Alfonso Cachero <alfonso.cachero@gmail.com>
Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Dmitry Petrov <dimakrm361@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Jan Krejčí <hokronx@gmail.com>
Co-authored-by: Kai Sommerfeld <kai.sommerfeld@gmx.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/am_et/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/be_by/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/el_gr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/fr_ca/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/hu_hu/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/is_is/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/si_lk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/zh_cn/
Translation: Kodi add-ons: PVR clients/pvr.hts

phunkyfish commented:
Ok, all the automation is in now @ksooo. When you get around to merging (no rush on this)  this another PR will be created for the release with the changelog entry. Then simply merge and release that.

It has worked consistently and well across all pvr addons so we could also automate the changelog and release step. In that case only this single PR would need to be merged.
